.github/workflows/Tests.yaml

```diff
@@ -38,6 +38,14 @@ on:
           Enter a version name YYYY.MM.DD[.TWEAK] to create a release on success
         required: false
         default: ''
+      timezone:
+        description: >
+          Timezone used for validating the version name. The release must be
+          approved by the end of the day in the specified timezone. See
+          /usr/share/zoneinfo for a list of possible values. Example:
+          America/Los_Angeles
+        required: false
+        default: 'UTC'
```

Comment:
Should we make the default `America/Los_Angeles`? The reason being that way unless someone starts the job at like 22:00 CET and doesn't specify the next day, it'll work. Phrased differently, that default covers the largest number of SXS people putting "today" as the date

Reply:
done